src/libData/AccountData/AccountStore.cpp

```diff
@@ -235,6 +238,90 @@ bool AccountStore::MoveUpdatesToDisk() {
   return true;
 }
 
+bool AccountStore::RepopulateStateTrie() {
+  LOG_MARKER();
+
+  unsigned int counter = 0;
+  bool batched_once = false;
+
+  vector<pair<Address, bytes>> entries;
+
+  for (const auto& i : m_state) {
```

Comment:
Is m_state thread safe? Now RepopulateStateTrie is running in a seprate thread

Reply:
Yes, the mutex was declared in the function where this was called